frontend-app/src/nikhil_ui_task/teacher/Teacher.js

```diff
@@ -0,0 +1,45 @@
+import React ,{useState} from 'react';
+import {Card ,Input ,Row, Col, Button} from 'antd';
+import TeacherTable from './TeacherTable';
+import { teachers } from '../data';
+
+const {Search} = Input;
+
+const Teacher = () => {
+    let [TeacherList,setTeacherList]=useState([...teachers]);
+
+    const onSearch = (value) => {
+      setTeacherList(value===""?teachers:teachers.filter(item =>
+         item.name.includes(value)||
+         item.subject.includes(value)||
+         item.email.includes(value)||
+         item.phone.includes(value)||
+         item.school.includes(value)
+       ));
+ };
+
+    const onClickRefresh =()=>{
+      new Promise((resolve,reject)=>setTimeout(()=>{setTeacherList([...teachers])},1000));
+    }
+
+  return (
+    <Card bordered= {true} className='card' hoverable={true}>
```

Comment:
Text of Table attributes can be made bold with some background to distinguish it as header